content/docs/command-reference/fetch.md

```diff
@@ -228,53 +228,51 @@ the cache entries for the `data/features` directory, and
 ## Example: With dependencies
 
 After following the previous example (**Specific stages**), only the files
-associated with the `prepare.dvc` stage file have been fetched. Several
+associated with the `prepare` stage have been fetched. Several
 dependencies/outputs of other pipeline stages are still missing from the cache:
 
 ```dvc
 $ dvc status -c
 ...
-    deleted:            model.pkl
     deleted:            data/features/test.pkl
     deleted:            data/features/train.pkl
-    deleted:            data/data.xml
+    deleted:            model.pkl
 ```
 
 One could do a simple `dvc fetch` to get all the data, but what if you only want
-to retrieve the data up to our third stage, `train.dvc`? We can use the
-`--with-deps` (or `-d`) option:
+to retrieve the data up to our third stage `train`? We can use the `--with-deps`
+(or `-d`) option:
 
 ```dvc
-$ dvc fetch --with-deps train.dvc
+$ dvc fetch --with-deps train
 
 $ tree .dvc/cache
 .dvc/cache
-├── 38
-│   └── 63d0e317dee0a55c4e59d2ec0eef33
-├── 42
-│   └── c7025fc0edeb174069280d17add2d4.dir
-├── 58
-│   └── 245acfdc65b519c44e37f7cce12931
-├── 68
-│   └── 36f797f3924fb46fcfd6b9f6aa6416.dir
-├── 9d
-│   └── 603888ec04a6e75a560df8678317fb
+├── 20
+│   └── b786b6e6f80e2b3fcf17827ad18597.dir
+├── 32
+│   └── b715ef0d71ff4c9e61f55b09c15e75
+├── 54
+│   └── c0f3ef1f379563e0b9ba4accae6807
+├── 6f
+│   └── 597d341ceb7d8fbbe88859a892ef81
+├── a1
+│   └── 414b22382ffbb76a153ab1f0d69241.dir
 ├── a3
-│   └── 04afb96060aad90176268345e10355
-├── aa
-│   └── 35101ce881d04b41d5b4ff3593b423
-└── dc
-    └── a9c512fda11293cfee7617b66648dc
+│   └── 04afb96060aad90176268345e10355
+└── c8
+    ├── 43577f9da31eab5ddd3a2cf1465f9b
+    └── d307aa005d6974a8525550956d5fb3
```

Comment:
> complete tree is displayed on line 251

BTW, do we need to display the whole thing? I'm not looking at the whole context, but please decide whether it's useful to have the full cache tree here or if it can be shortened and why. Thanks

Reply:
I think the complete tree is displayed here because in previous example about prepare stage, the tree displayed only 3 values. But in this  example we are fetching remaining stages. So it shows that more files are downloaded and hence cache contents increased. We can display it like this also:
```
.dvc/cache
...
├── a1
│   └── 414b22382ffbb76a153ab1f0d69241.dir
├── a3
│   └── 04afb96060aad90176268345e10355
└── c8
    ├── 43577f9da31eab5ddd3a2cf1465f9b
    └── d307aa005d6974a8525550956d5fb3
```
Only the last 3 files.

Reply:
Yeah, looks better but why only those last 3 though? Are those the only values actually used throughout the examples? If so, I agree.

Reply:
Only `c8....` and one more file `20b7...` is used in first example.
I think it's better to display whole tree as the purpose of the tree was to display that more files are downloaded and hence cache contents are increased.
Else we can display these two files above and one more random file like this:
```
....
├── a1
│   └── 414b22382ffbb76a153ab1f0d69241.dir
├── 20
│   └── b786b6e6f80e2b3fcf17827ad18597.dir
└── c8
    ├── 43577f9da31eab5ddd3a2cf1465f9b
    └── d307aa005d6974a8525550956d5fb3
```

Reply:
OK